bobtask/task.go

```diff
@@ -94,6 +95,18 @@ type Task struct {
 
 	// skippedInputs is a lists of skipped input files
 	skippedInputs []string
+
+	Dependencies []string `yaml:"dependencies"`
+
+	// Contains Dependencies + its bobfile Dependencies
+	AllDependencies []string
+
+	// Paths from /nix/store computed from all its dependencies
+	StorePaths []string
+	UseNix     bool
+	Nixpkgs    string
+
+	listRecursiveCache map[string][]string
```

Comment:
Actually i think the global one made more sense. As implemented now it will use a different cache for each run. If you need to diable the cache for each run i a test.. add a option to disable caching or just reinitialise bob on each test.